channels:
  - conda-forge
dependencies:
  - scipy
  - s3fs>=2022.1.0
  - matplotlib<=3.6.1
  - openjdk=11
  - cmake
  - make
  - gcc_linux-64<9.0
  - gxx_linux-64
  - scikit-learn==1.1.*
  - sqlalchemy
  - snowflake-sqlalchemy
  - snowflake-connector-python
  - psycopg2
  - pymysql
  - ipyparallel==8.6.1
  - ipywidgets==8.1.1
  - ipython=8.16.1
  - nbconvert=6.5.0
  - pandas >=2,<2.1
  - "_openmp_mutex=*=*gnu*"
  - bodo.ai::bodo-azurefs-sas-token-provider=1.0
  #NOTE: ideally, this would just come from the Bodo dependencies
  #However, we encounter a wierd error if we do it at that point
  - pyarrow=13.0.0
  - pip
  - pip:
      - httpie
      - bodo-platform-ipyparallel-kernel==2.4.0
